src/app/api/deleteTask/[id]/route.ts

```diff
@@ -0,0 +1,22 @@
+import prisma from "@/lib/prisma";
+
+export async function DELETE(
+  request: Request,
+  { params }: { params: { id: string } }
+) {
+  try {
+    await prisma.task.delete({
+      where: {
+        id: parseInt(params.id, 10),
+      },
+    });
+
+    return new Response(JSON.stringify("this task deleted successfully"), {
+      status: 200,
+    });
+  } catch (error) {
+    return new Response(JSON.stringify({ error: (error as Error).message }), {
```

Comment:
**Suggestion: sometimes you don't want to return the actual error to the user. "An error happened" or something vague is maybe better**

Reply:
**Suggestion: better to log the error as well, if you are going to try/catch the error.**

**Question: do you ever need the try/catch? Why not just omit it?**

Reply:
done